### PR TITLE
optimize index merge

### PR DESCRIPTION
**Pull request by binlijin**
Relative to  https://github.com/druid-io/druid/pull/1938

### CODE REVIEW

**Comment by xvrl** [comment]
:+1:

**Comment by gianm** [comment]
:+1:

File: processing/src/test/java/io/druid/segment/QueryableIndexIndexableAdapterTest.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+	   
+  @Test
+  public void testGetBitmapIndexSeeker() throws Exception
+  {
+    final long timestamp = System.currentTimeMillis();
+    IncrementalIndex toPersist = IncrementalIndexTest.createIndex(null);
+    IncrementalIndexTest.populateIndex(timestamp, toPersist);
+
+    final File tempDir = temporaryFolder.newFolder();
+    final IndexableAdapter incrementalAdapter = new IncrementalIndexAdapter(
```
Comment (xvrl) [inline processing/src/test/java/io/druid/segment/QueryableIndexIndexableAdapterTest.java]: incrementalAdapter doesn't seem to be used anywhere.

Comment (binlijin) [inline processing/src/test/java/io/druid/segment/QueryableIndexIndexableAdapterTest.java]: Yes, my mistake, this is useless.

**Pull request merged**